File: Udemy/Algoritmos e Logica de Programacao/C/Estrutura Sequencial/ExerciciosEstruturaSequencial/ProblemaRetangulo.c
/*Problema "retangulo" Fazer um programa para ler as medidas da base e altura de um retângulo.
Em seguida, mostrar o valor da área, perímetro e diagonal deste retângulo,
com quatro casas decimais, conforme exemplos. */

#include<stdio.h>
#include<stdlib.h>
#include<math.h>

int main(){

    double base, altura, area, perimetro, diagonal;

    printf("Base do retangulo: ");
    scanf("%lf",&base);

    printf("Altura do retangulo: ");
    scanf("%lf",&altura);

    area = base * altura;

    perimetro = 2 * (base + altura);

    diagonal = sqrt(pow(base,2)+pow(altura,2));

    printf("AREA = %.4lf\n",area);
    printf("PERIMETRO = %.4lf\n",perimetro);
    printf("DIAGONAL = %.4lf",diagonal);


    return 0;
}

File: Udemy/Algoritmos e Logica de Programacao/Python/Estrutura Sequencial/ProblemaRetangulo.py
'''Problema "retangulo"
Fazer um programa para ler as medidas da base e altura de um retângulo. Em seguida, mostrar o valor
da área, perímetro e diagonal deste retângulo, com quatro casas decimais, conforme exemplos.'''

import math

base: float; altura: float; area: float; perimetro: float; diagonal: float;

base = float(input("Base do retangulo: "))
altura = float(input("Altura do retangulo:"))
area = base * altura
perimetro = 2 * (base + altura)
diagonal = math.sqrt(pow(base,2)+pow(altura,2))

print(f"AREA = {area:.4f}")
print(f"PERIMETRO = {perimetro:.4f}")
print(f"DIAGONAL = {diagonal:.4f}")